src/components/Blog/Post/Markdown/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
```

Comment:
Not sure why, but `external-link` elements don't seem to be working: 

<img width="1580" alt="image" src="https://user-images.githubusercontent.com/43496356/160920986-36bd2364-8359-4e1d-8f26-6de28dfe002a.png">

Reply:
These got scooped up into the theme, but the plugin's still added into the remark definition so it's unfortunately not as easy as the plugin not being added. I'll have to look into it more!

Reply:
[Fixed![(https://dvc-org-try-external-th-doe8zu.herokuapp.com/blog/march-22-heartbeat#march-office-hours)

Basically, the issue stems from me upgrading all our packages in this PR (probably unnecessary looking back on it, but we'd have to eventually anyway)

Most packages in the `unified` family like `remark`, `rehype` and derivatives were changed to only export ESM modules, preventing us from importing them with `require` in non-webpacked parts of the application like `gatsby-node` and `gatsby-config` (TypeScript support didn't help here unfortunately).

What does work is dynamic `import` calls, but that returns `Promise`s and the `import` calls can't be at the top level; by extension, all functions that consume them must be turned asynchronous.

And that's what I did! Our custom remark plugins (these elements as well as the image resizing plugin) are now all async functions which use dynamic imports for the packages that need them. Took a bit of refactoring and I had some stumbles along the way, but it seems they work just as well as before now and are more future-proofed.

TL;DR: updating remark broke our remark plugins, and I just finished up a moderately large refactor that was needed to get them working again.